<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>save_images</class>
 <widget class="QDialog" name="save_images">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Images</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="all_detectors">
     <property name="text">
      <string>All Detectors</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">detector_selection</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QRadioButton" name="single_detector">
     <property name="text">
      <string>Single Detector</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">detector_selection</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="detectors">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="directory_label">
     <property name="text">
      <string>Directory:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="pwd">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="change_directory">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLineEdit" name="file_stem">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The detector name will be appeneded to the file stem on save.&lt;br/&gt;Example:&lt;/p&gt;&lt;p&gt;FF_Ruby_000039-fc.npz -&amp;gt; FF_Ruby_000039-fc_ge1.npz&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>ex: FF_Ruby_000039-fc</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="format">
     <item>
      <property name="text">
       <string>fch5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>npz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hdf5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="ignore_agg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Ignore Aggregation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>all_detectors</tabstop>
  <tabstop>single_detector</tabstop>
  <tabstop>detectors</tabstop>
  <tabstop>pwd</tabstop>
  <tabstop>change_directory</tabstop>
  <tabstop>file_stem</tabstop>
  <tabstop>format</tabstop>
  <tabstop>ignore_agg</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>save_images</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>save_images</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="detector_selection"/>
 </buttongroups>
</ui>
